src/main/java/frc/robot/components/TrigonPlaystationController.java

```diff
@@ -142,37 +142,23 @@ public void setRumble(double power) {
      */
     public void intermittentRumble(int quantity) {
         rumbleAmount = quantity * 2 - 1;
-        notifier.startPeriodic(kIntermittentRumbleTime);
+        notifier.startPeriodic(RobotConstants.DriverConstants.RUMBLE_INTERMISSION_TIME);
     }
 
-    /**
-     * Get the X axis value of the controller.
-     *
-     * @param hand Side of controller whose value should be returned.
-     * @return The X axis value of the controller.
-     */
-    @Override
-    public double getX(Hand hand) {
-        if(hand.equals(Hand.kLeft)) {
-            return getRawAxis(Axis.leftX.getAxis());
-        } else {
-            return getRawAxis(Axis.rightX.getAxis());
-        }
+    public double getLeftX() {
+        return getRawAxis(Axis.leftX.getAxis());
     }
 
-    /**
-     * Get the Y axis value of the controller.
-     *
-     * @param hand Side of controller whose value should be returned.
-     * @return The Y axis value of the controller.
-     */
-    @Override
-    public double getY(Hand hand) {
-        if(hand.equals(Hand.kLeft)) {
-            return getRawAxis(Axis.leftY.getAxis());
-        } else {
-            return getRawAxis(Axis.rightY.getAxis());
-        }
```

Comment:
why delete? both this and the new function can work

Reply:
There's no `Hand` class anymore